renormalizer/model/basis.py

```diff
@@ -192,6 +192,8 @@ def op_mat(self, op: Union[Op, str]):
                                          )
             else:
                 mat = np.diag(self.dvr_x**2)
+        elif op_symbol == "x x":
```

Comment:
Cases like this can be simplified by:
```
elif op_symbol == "x^2" or op_symbol=="x x":
```

Reply:
It's because another flag self.general_xp_power would lead to inconvenience.